apps/backend/src/modules/meetings/meeting.service.ts

```diff
@@ -0,0 +1,127 @@
+import { MeetingError } from "@meetlytic/shared";
+
+import { HTTPCode } from "~/libs/modules/http/http.js";
+import { type Service } from "~/libs/types/types.js";
+
+import { MeetingStatusMessage } from "./libs/enums/enums.js";
+import {
+	type MeetingCreateRequestDto,
+	type MeetingGetAllResponseDto,
+	type MeetingResponseDto,
+	type MeetingUpdateRequestDto,
+} from "./libs/types/types.js";
+import { MeetingEntity } from "./meeting.entity.js";
+import { type MeetingRepository } from "./meeting.repository.js";
+
+class MeetingService implements Service<MeetingResponseDto> {
+	private meetingRepository: MeetingRepository;
+
+	public constructor(meetingRepository: MeetingRepository) {
+		this.meetingRepository = meetingRepository;
+	}
+
+	public async create(
+		payload: MeetingCreateRequestDto,
+	): Promise<MeetingResponseDto> {
+		const meeting = MeetingEntity.initializeNew({
+			host: payload.host,
+			instanceId: payload.instanceId ?? null,
+			ownerId: payload.ownerId,
+		});
+
+		const newMeeting = await this.meetingRepository.create(meeting);
+		return newMeeting.toObject();
+	}
+
+	public async delete(id: number, ownerId?: number): Promise<boolean> {
+		const meeting = await this.meetingRepository.find(id);
+		if (!meeting) {
+			throw new MeetingError(
+				MeetingStatusMessage.MEETING_NOT_FOUND,
+				HTTPCode.NOT_FOUND,
+			);
+		}
+
+		if (ownerId !== undefined && meeting.toObject().ownerId !== ownerId) {
+			throw new MeetingError(
+				MeetingStatusMessage.FORBIDDEN,
+				HTTPCode.FORBIDDEN,
+			);
+		}
+
+		return await this.meetingRepository.delete(id);
+	}
+
+	public async find(id: number, userId?: number): Promise<MeetingResponseDto> {
```

Comment:
```suggestion
	public async find(id: number): Promise<MeetingResponseDto> {
```
All additional checks should be done using hooks